parquet-protobuf/pom.xml

```diff
@@ -32,7 +32,7 @@
   <properties>
     <elephant-bird.version>4.4</elephant-bird.version>
     <protobuf.version>3.25.5</protobuf.version>
-    <common-protos.version>2.41.0</common-protos.version>  <!-- make sure it's compatible with protobuf.version -->
+    <common-protos.version>2.50.0</common-protos.version>  <!-- make sure it's compatible with protobuf.version -->
```

Comment:
It is compatible: https://mvnrepository.com/artifact/com.google.api.grpc/proto-google-common-protos/2.50.0